kratos/geometries/plane_3d.h

```diff
@@ -0,0 +1,274 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Pooyan Dadvand
+//  contributors:    Ruben Zorrilla
+//
+
+#if !defined(KRATOS_PLANE_3D_H_INCLUDED)
+#define  KRATOS_PLANE_3D_H_INCLUDED
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "geometries/point.h"
+#include "includes/checks.h"
+#include "utilities/math_utils.h"
+
+namespace Kratos
+{
+
+///@name Kratos Globals
+///@{
```

Comment:
I would remove all the doxygen empty grouping comments in order to reduce the size of the header and have less scroll to see the real code.

Reply:
Done!